instagram/src/components/CommentSection/CommentSection.js

```diff
@@ -0,0 +1,83 @@
+import React from 'react';
+import styled from 'styled-components';
+import CommentForm from './CommentForm';
+import PropTypes from 'prop-types';
+import moment from 'moment';
+
+const IndividualComment = styled.div`
+    margin: 10px 14px;
+    font-size: 1.6rem;
+
+    span {
+        font-weight: bold;
+    }
+`
+
+const Timestamp = styled.p`
+    margin: 14px;
+    font-size: 1.1rem;
+`
+
+
+
+class CommentSection extends React.Component {
+    constructor(props) {
+        super(props);
+        this.state = {
+            comments: this.props.comments,
+            timestamp: this.props.postTimestamp,
+            newComment: '',
+        }
+    }
+
+    addNewComment = event => {
```

Comment:
Writing functionality this low into the component tree is questionable as it makes the application more brittle by violating the single source of truth.  I would try to lift this state to App.js and control the comment sections from there